data/lib/src/network/linshare_http_client.dart

```diff
@@ -285,4 +286,16 @@ class LinShareHttpClient {
 
     return SharedSpaceNodeNestedResponse.fromJson(deletedSharedSpace);
   }
+
+  Future<WorkGroupNodeFolderDto> createSharedSpaceNodeFolder(String name, SharedSpaceId sharedSpaceId, {parent = WorkGroupNodeId}) async {
+    final createSharedSpaceNodeRequest = CreateSharedSpaceNodeFolderRequest(name, parentWorkGroupNodeId: parent);
+    final resultJson = await _dioClient.post(
+      Endpoint.sharedSpaces
+        .withPathParameter(sharedSpaceId.uuid)
+        .withPathParameter('nodes')
+        .withPathParameter(parent?.uuid)
```

Comment:
Have you tested the case `parent` is `null` ?

Reply:
Please focus to test the case `parent` is `null`